src/explore-education-statistics-admin/package.json

```diff
@@ -40,6 +40,7 @@
     "fs-extra": "^9.1.0",
     "geojson": "^0.5.0",
     "govuk-frontend": "^4.7.0",
+    "html-react-parser": "^1.3.0",
```

Comment:
We shouldn't need this if we just use the DOM to detect invalid images (see other comment)